CMakeLists.txt

```diff
@@ -92,7 +83,7 @@ if(BUILD_SHARED_LIBS)
 
 
 	if(DEFINED CMAKE_VERSION AND NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.11")
-		target_include_directories(TinyEXIF PUBLIC 
+		target_include_directories(TinyEXIF PUBLIC
```

Comment:
Avoid any extra unnecessary change in the file, we should have the minimal number of changes with upstream. Even if there are whitespaces.

Reply:
My text editor is set to automatically remove unnecessary whitespace.
I really don't think it's convenient to have to -before the 'push'- deactivate this feature, add the empty spaces that the editor automatically deleted, do the 'push', and then reactivate the feature.

Reply:
You can control what change you stage/include in the commit. I'll try to take the issue from here do we can include the submodule.